lib/jsonapi/serializable/conditional_fields.rb

```diff
@@ -0,0 +1,65 @@
+module JSONAPI
+  module Serializable
+    module ConditionalFields
+      def self.prepended(klass)
+        klass.class_eval do
+          extend ClassMethods
+          prepend InstanceMethods
+          class << klass
+            attr_accessor :condition_blocks
+          end
+          self.condition_blocks = {}
+        end
+      end
+
+      module ClassMethods
+        def _register_condition(name, conditions)
```

Comment:
I get the `_`, but probably should also mark this and other `_` methods as private, yeah?